database/src/main/resources/db/migration/V1_162__create_interface_download_table.sql

```diff
@@ -0,0 +1,29 @@
+-- We need to store interface download state (success, failure) and errors to db.
+-- This table is reserved for that.
+
+-- Create state enum
+CREATE TYPE interface_download_status AS ENUM ('success','failure');
+
+-- And create the table
+CREATE TABLE "interface-download" (
```

Comment:
This table is connected with `external-interface-description`, it might be a better human-readable convention to name it with same prefix. E.g. `external-interface-status` or something better.